docs/docsite/rst/dev_guide/core_index.rst

```diff
@@ -56,7 +56,7 @@ Find the task that best describes what you want to do:
 
   * I want to :ref:`understand how to contribute to Ansible <ansible_community_guide>`.
   * I want to :ref:`contribute my module or plugin <developing_modules_checklist>`.
-  * I want to :ref:`understand the license agreement <contributor_license_agreement>` for contributions to Ansible.
+  * I want to :ref:`understand the DCO agreement <developer_certificate_of_origin>` for contributions to Ansible.
```

Comment:
Assuming core agrees to this, I'd change this text and the rest of the places it is repeated to something like

```suggestion
  * I want to :ref:`understand the DCO agreement <developer_certificate_of_origin>` for contributions to `ansible-core <https://github.com/ansible/ansible>`_ and the `Ansible Documention <https://github.com/ansible/ansible-documentation>`_ repositories.
```

Reply:
OK, we could use this for now. However, I'd like to think about how that's going to read when we're approaching 40%, 50%, 60% of the project signed up to the DCO. We'll end up with frequent updates here and a *very* long line - so we need to think about when it's acceptable to say "Ansible".

As an alternative, consider that we could use wording like "I want to understand the DCO agreement for contributions to Ansible projects (where applicable, see the CONTRIBUTING in project repo)" which then gives a carve out for those who haven't got there yet, and needs less updating - how does that feel?

Reply:
Something to keep in mind - these docs are only applicable to ansible-core and collections. Dev guides for every other Ansible ecosystem project is handled by those projects, not this doc.

Reply:
Ah OK, that helps. In that case I'm OK with the change if you are @samccann?

Reply:
yep sounds good to me

Reply:
> Ah OK, that helps. In that case I'm OK with the change if you are @samccann?


To be clear, does `the change` refers to my suggestion or @GregSutcliffe's?

Reply:
I'm saying I'm happy with your proposed alteration @gotmax23.